src/components/ha-button.ts

```diff
@@ -16,12 +16,15 @@ export class HaButton extends Button {
       }
       .mdc-button {
         height: var(--button-height, 36px);
+        text-align: var(--button-text-align, center);
```

Comment:
```suggestion
        text-align: var(--ha-button-text-align, center);
```

When you introduce this new css prop I suggest to use `<component-name>-<prop>`

I know it's not done everywhere like this, but we would like to use this format.